commit fab33ae9aae39e66824ea59cc2385a027695b38b
Author: Alex Wilson <alex@uq.edu.au>
Date:   2019-08-12T03:54:31+00:00 (8 weeks ago)
    
    joyent/node-mahi#2 `Client#getUserById` always calls its callback twice
    Reviewed by: Robert Mustacchi <rm@joyent.com>
    Approved by: Robert Mustacchi <rm@joyent.com>

diff --git a/lib/client.js b/lib/client.js
index 9f3f105..363df52 100644
--- a/lib/client.js
+++ b/lib/client.js
@@ -346,7 +346,6 @@ MahiClient.prototype.getUserById = function getUserById(uuid, cb) {
     var self = this;
     var fmt = '/users/%s';
     var path = sprintf(fmt, uuid);
-    self._get(path, cb);
     self._get(path, function gotUser(err, info) {
         if (err) {
             cb(err);
diff --git a/package.json b/package.json
index 8cb674c..9a70b22 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
     "name": "mahi",
     "description": "Mahi client",
-    "version": "2.3.0",
+    "version": "2.3.1",
     "author": "Joyent (joyent.com)",
     "main": "index.js",
     "dependencies": {
